contrib/mingw64.cross

```diff
@@ -0,0 +1,16 @@
+# dnf install mingw64-gcc mingw64-pkg-config mingw64-glib2 mingw64-libgusb mingw64-sqlite mingw64-libarchive mingw64-json-glib mingw64-libsoup
+# meson .. --prefix=~/.root --cross-file=../contrib/mingw64.cross -Dbuild=standalone -Dgpg=false -Dplugin_uefi=false -Dplugin_redfish=false -Dplugin_altos=false -Dplugin_dell=false -Dplugin_nvme=false -Dsystemd=false -Dplugin_emmc=false -Dplugin_amt=false -Dintrospection=false -Dplugin_thunderbolt=false -Dplugin_synaptics=false -Dman=false -Dgcab:introspection=false -Dlibxmlb:introspection=false -Dgudev=false
+
```

Comment:
Can you add a CI target with tests off calling this?

Reply:
I see you're on the case, thanks!